src/utils/error.js

```diff
@@ -0,0 +1,10 @@
+export const ERROR_STRINGS = {
+  FIELD: {
+    NAME_REQUIRED: "Field components require a name",
```

Comment:
We don't have such term as "field components", even though one might guess what it means.
A more useful message would be e.g.:
> Input component is missing a `name` prop

For this, we need to know the caller component name.
Looks like there is no nice way to get the caller name when using a strict mode.
I'm thinking that passing the caller name as an argument to `useField` is probably the safest way to go, e.g.:

```
// Checkbox.js

const ... = useField("Checkbox", { name, ... });
```

What do you think?

Reply:
Yeah I was actually going to chat to you about this one, is definitely better to have more specific error messages. I can't think of any better way to do it either so I think it's a good idea, might even come in handy later down the line. I'll update it :)